osi_detectedlandmark.proto

```diff
@@ -24,164 +24,219 @@ message DetectedTrafficSign
     //
     optional Identifier ground_truth_id = 3;
 
-    // A list of candidates for this traffic sign as estimated by the sensor.
-    //    
-    repeated CandidateSign candidate_sign = 4;
-
-    // A list of candidates for (a) possible supplementary sign(s) as estimated
-    // by the sensor.
+    // The main sign as estimated by the sensor.
     //
-    repeated CandidateSupplementarySign candidate_supplementary_sign = 5;
+    optional EstimatedMainSign main_sign = 4;
 
-    // The detected geometry of the traffic sign.
-    //    
-    optional Geometry geometry = 6;
+    // A list of additional supplementary sign(s) as estimated by the sensor.
+    //
+    // \note OSI uses singular instead of plural for repeated field names.
+    //
+    repeated EstimatedSupplementarySign supplementary_sign = 5;
```

Comment:
We have never used the word "estimated" before. It should be "detected"

Reply:
We get a problem, if we use the message structure for the detected messages. 

Would you prefer DetectedSupplementarySignData?